### PR TITLE
Add Face Key-Points

### PR DESCRIPTION
**Pull request by tomer-algo**
# Intro
  - Add Face Key-Points to Skeleton Model
# Project
  - Skeleton
  - Pose 
# Implemented Sections
  - Add Face Key-Points to Skeleton Model
    - Training
    - Evaluation
    - Analysis
  - Create New 'init' model with Face Key-Points
  - Adjust Export
# Tests Description
  - Run Training https://wandb.ai/algo/auto-skeleton/runs/1qpc7zn6?workspace=user-tomershmul
  - Export Model + TorchServe Inference
  - Run Analysis

x: GT ; o: Prediction
0be92432_2020_11_13_446_homography

# Undeveloped Sections
  - 
# What's New
  - Face Key-Points
# Checklist
  - [x] Module tested fully after last commit
  - [ ] Scheduled a PR meeting (if needed)
  - [ ] Documentation (comments or a dedicated documnet)
### Tested platforms
  - [x] Local
  - [ ] Cloud (Sagemaker)
  - [ ] Staging
  - [ ] Not relevant
### Tested Cameras
  - [ ] Gen 1.5
  - [ ] Gen 2
  - [x] Not relevant
# Comments
### Included Features
  - 
### Infrastructure Dependencies
  - Use s3://nanit-algo-resources/algo-infra/package-versions/nanit_algo_infra_online_additions-1.1.2-py3-none-any.whl
### Tech Comments
  - 
### Testing Instructions
  - 
### Todos
  - Add Face Key-Points to Pose (Another PR)

### CODE REVIEW

File: lib/models/pose_hrnet.py [hunk 1]
```diff
@@ -547,6 +549,35 @@ def handle_heel_init_if_needed(self, need_init_state_dict):
...
+        face_weights = torch.zeros(face_weights_shape)
+        face_biases = torch.zeros(len(coco_face_joints_mapping.keys()))
+
+        for i, coco_index in enumerate(coco_face_joints_mapping.values()):
+            face_weights[i] = model_coco.final_layer.weight[coco_index].clone().unsqueeze(0)
+            face_biases[i] = model_coco.final_layer.bias[coco_index].clone()
+
+        need_init_state_dict['final_layer.weight'] = torch.cat((need_init_state_dict['final_layer.weight'], face_weights), dim=0)
+        need_init_state_dict['final_layer.bias'] = torch.cat((need_init_state_dict['final_layer.bias'], face_biases), dim=0)
+
```
Comment (felixNan) [inline lib/models/pose_hrnet.py:580]: Could you please explain what should this function do?

Comment (tomer-algo) [reply lib/models/pose_hrnet.py:580]: this code is used once to create a model to load weights from him to start skeleton training for all joints (incl. face Key-Points)

Comment (felixNan) [reply lib/models/pose_hrnet.py:580]: You just add a face branch to basic skeleton model and initialize it with COCO?

Comment (tomer-algo) [reply lib/models/pose_hrnet.py:580]: yes
and then I've trained it on our data

**Pull request merged**